### PR TITLE
Githubauth

### PR DESCRIPTION
**Pull request by jkuruzovich**
This updates the Jupyterhub image in order to enable GitHub authorization.

Should close https://github.com/jupyterhub/helm-chart/issues/30 once the standard images are rebuilt.

### CODE REVIEW

File: jupyterhub/templates/hub.yaml [hunk 1]
```diff
@@ -66,6 +66,12 @@ data:
   auth.google.login-service: {{.Values.auth.google.loginService | quote}}
   auth.google.callback-url: {{.Values.auth.google.callbackUrl}}
   {{- end }}
+  {{ if eq .Values.auth.type "github" -}}
+  auth.github.client-id: {{.Values.auth.github.clientId | quote}}
+  auth.github.client-secret: {{.Values.auth.github.clientSecret | quote}}
+  auth.github.callback-url: {{.Values.auth.github.callbackUrl}}
```
Comment (yuvipanda) [inline jupyterhub/templates/hub.yaml]: should pass this to quote too!

**Comment by yuvipanda** [comment]
Thank you!

**Comment by jkuruzovich** [comment]
Thank you! That was quick!

**Comment by yuvipanda** [comment]
:D

**Pull request merged**